Docs/source/running_cpp/parameters.rst

```diff
@@ -28,6 +28,15 @@ Overall simulation parameters
     The direction of the Lorentz-transform for boosted-frame simulations
     (The direction ``y`` cannot be used in 2D simulations.)
 
+* ``warpx.zmax_plasma_to_compute_max_step`` (`float`) optional (default `-inf`)
+    Used when running in a boosted frame. If specified, automatically 
```

Comment:
Could you remove `(default -inf)`?
Also should we replace `Used when running...` by `Can be useful when running...`, just to emphasize that this is optional?

Reply:
Sure, this was included in the latest commit